docs/develop/api-reference/list-collections.md

```diff
@@ -13,12 +13,14 @@ This page details the Momento API methods for the [list collection data types](.
 
 ### ListFetch
 
-Gets a list item from a cache.
+Gets a list item from a cache, with optional slices.
 
-| Name            | Type   | Description                                   |
-| --------------- | ------ | --------------------------------------------- |
-| cacheName       | String | Name of the cache.                            |
-| listName        | String | The name of the list item to be retreived.         |
+| Name       | Type   | Description                                                                 |
+|------------|--------|-----------------------------------------------------------------------------|
+| cacheName  | String | Name of the cache.                                                          |
+| listName   | String | The name of the list item to be retrieved.                                  |
+| startIndex | Number | The starting inclusive element of the list to fetch. Default is 0.          |
+| endIndex   | Number | The ending exclusive element of the list to fetch. Default is end of list.  |
```

Comment:
do we have any convention for indicating these are optional?  Or should we add (optional) in the description?

Reply:
Which ones are optional?